apps/cookbook/src/app/examples/calendar-example/calendar-card-example.component.ts

```diff
@@ -61,26 +66,14 @@ export class CalendarCardExampleComponent implements OnChanges {
   }
 
   private updateInputDates() {
-    const todayMoment = (this.useTimezoneUTC ? moment.utc() : moment()).startOf('day');
+    const today = this.useTimezoneUTC ? getUtcDate(startOfDay(new Date())) : startOfDay(new Date());
 
-    this.minDate = todayMoment
-      .clone()
-      .subtract(60, 'days')
-      .toDate();
-    this.maxDate = todayMoment
-      .clone()
-      .add(60, 'days')
-      .toDate();
-    this.todayDate = todayMoment
-      .clone()
-      .add(3, 'days')
-      .toDate(); // artificial but works for demo
+    this.minDate = subDays(today, 60);
+    this.maxDate = addDays(today, 60);
+    this.todayDate = addDays(today, 3); // artificial but works for demo
 
     this.disabledDates = [3, 5, 7, 10, 15, 25, 28, 35].map((daysFomToday) =>
-      todayMoment
-        .clone()
-        .add(daysFomToday, 'days')
-        .toDate()
+      addDays(today, daysFomToday)
```

Comment:
Boyscout suggestion: Fix the typo in `daysFomToday` 😊

Reply:
![giphub](https://media2.giphy.com/media/QhmboW0R7eUbm/giphy.gif)